### ISSUE TITLE
TypeScript version update - 1.6.2

### ISSUE THREAD

**Issue by mickeyvip** [ArpNetworking/sbt-typescript#16]
Hello.

Can you update the TypeScript version to `1.6.2`, that was released recently?

The WebJars repo of `typescript-node` has been updated also.

Thank you.

**Comment by BrandonArp** [comment]
Thanks for the issue.  I've released a new version (0.1.9) to central that uses typescript 1.6.2 compiler.

**Comment by mickeyvip** [comment]
Although the new version uses `1.6.2` compiler - it lacks new features.

One of them is more module formats:

https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L2073

``` javascript
export const enum ModuleKind {
    None = 0,
    CommonJS = 1,
    AMD = 2,
    UMD = 3,
    System = 4,
}
```

Also, it seems from their example that you don't need to use numeric constants, there is an Enum for modules:

``` javascript
import * as ts from "typescript";
/// ...
let commonJsModule = ts.ModuleKind.CommonJS;
let systemModule = ts.ModuleKind.System;
```

Thank you.
